React/Views/RCTRefreshControl.m

```diff
@@ -52,18 +57,9 @@ - (void)beginRefreshing
 {
   // When using begin refreshing we need to adjust the ScrollView content offset manually.
   UIScrollView *scrollView = (UIScrollView *)self.superview;
-  CGPoint offset = {scrollView.contentOffset.x, scrollView.contentOffset.y - self.frame.size.height};
-
-  // `beginRefreshing` must be called after the animation is done. This is why it is impossible
```

Comment:
before, the scrollView would move down with animation and *after* that the refreshControl would show up. I used the refreshControl to indicate fetching in progress. If the fetching was really fast, you'd only see the scrollView move down and then up, without the refreshControl becoming visible.
Now the scrollView moves down with animation and the refreshControl is shown even before the animation is done, so the refreshControl is visible even when the fetching is really fast.
I have only tested this on emulator. Will test it on a device as well and let you know.